src/dlgConnectionProfiles.cpp

```diff
@@ -175,34 +173,69 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
         port_ssl_tsl->setEnabled(false);
     }
 
-    mRegularPalette.setColor(QPalette::Text, QColor(0, 0, 192));
-    mRegularPalette.setColor(QPalette::Highlight, QColor(0, 0, 192));
-    mRegularPalette.setColor(QPalette::HighlightedText, QColor(Qt::white));
-    mRegularPalette.setColor(QPalette::Base, QColor(Qt::white));
+    profiles_tree_widget->setViewMode(QListView::IconMode);
+    this->setStyleSheet(mStyleSheetRO);
+}
+
+void dlgConnectionProfiles::removeNotification(QString notif)
+{
+    QString msg; //="<html><head/><body>";
 
-    mReadOnlyPalette.setColor(QPalette::Base, QColor(212, 212, 212));
-    mReadOnlyPalette.setColor(QPalette::Text, QColor(0, 0, 192));
-    mReadOnlyPalette.setColor(QPalette::Highlight, QColor(0, 0, 192));
-    mReadOnlyPalette.setColor(QPalette::HighlightedText, QColor(Qt::white));
+    mErrorList.removeDuplicates();
+    QString str = QString("^%1.+").arg(notif.left(20));
+    int index = mErrorList.indexOf(QRegExp(str));
```

Comment:
IIIRC `QRegExp` is obsolete, we should be using `QRegularExpression` instead I believe... :worried: